src/quo/components/tags/collectible_tag/style.cljs

```diff
@@ -0,0 +1,50 @@
+(ns quo.components.tags.collectible-tag.style
+  (:require
+    [quo.foundations.colors :as colors]))
+
+(defn container
+  [size options blur? theme]
+  (let [hold? (= options :hold)]
+    (merge {:background-color (if blur?
+                                (colors/theme-colors colors/neutral-80-opa-5 colors/white-opa-5 theme)
+                                (colors/theme-colors colors/neutral-10 colors/neutral-90 theme))
+            :flex-direction   :row
+            :align-items      :center
+            :padding-left     2
+            :border-width     (if hold? 1 0)
+            :border-radius    0
+            :border-color     (colors/theme-colors colors/success-50 colors/success-60 theme)}
+           (condp = size
+             :size-24 {:height        (if hold? 26 24)
+                       :padding-right 10
+                       :border-radius (if hold? 8 8)}
```

Comment:
this does not look necessary

Reply:
true, I was experimenting and forgot to remove it. Thank you